### DISCUSSION TITLE
GitHub Copilot is already enabled but I can't use

### DISCUSSION THREAD

**Discussion by Afacanc38** [community/community#15791] [Copilot Conversations] [answered]
GitHub Copilot is enabled on my account.
![Screenshot 2022-04-29 at 14-43-44 Build software better together](https://user-images.githubusercontent.com/66299502/165938191-550ef437-7597-4245-ac19-95ecc286367d.png)
But after installing the plugin and entering the access token, I got such an error.
Note: I am using VSCodium.
![image](https://user-images.githubusercontent.com/66299502/165938455-0450c882-b61b-45e4-8fa1-72757a54bb62.png)
![Ekran görüntüsü 2022-04-29 14-46-06](https://user-images.githubusercontent.com/66299502/165938484-653afaef-cdfa-4f00-9b5d-ba35f5f00192.png)

**Comment by davecheney** [comment, marked as answer]
Currently copilot doesn’t work with forks like VScodium. Please download vscode.
